src/wasm.rs

```diff
@@ -1,10 +1,13 @@
-use alloc::vec::Vec;
+use crate::alloc::vec::Vec;
+#[cfg(feature = "wasm")]
```

Comment:
This file is only compiled when the wasm feature is used, so this line is not necessary.

Reply:
You're right. Fixed.